src/mpr/modules/utility/oleToDate.ts

```diff
@@ -0,0 +1,8 @@
+// distance from ole to epoc in milliseconds
+export function oleToUTCDate(ole: number): Date {
+  const oleToEpoch = new Date('12/30/1899 00:00:000Z').getTime();
```

Comment:
```suggestion
  const oleToEpoch = new Date('1899-12-30T00:00:00.000Z').getTime();
```

Use standard ISO string.
I also suggest to move it to a top-level constant.